doc/source/computation.rst

```diff
@@ -47,17 +48,18 @@ NA/null values *before* computing the percent change).
 Covariance
 ~~~~~~~~~~
 
-The ``Series`` object has a method ``cov`` to compute covariance between series
-(excluding NA/null values).
+The ``Series`` object has a method :meth:`~Series.cov` to compute covariance 
```

Comment:
How about

    :meth:`Series.cov` can be used to compute covariance between series (excluding missing values)